include/rtcdcpp/PeerConnection.hpp

```diff
@@ -60,25 +59,32 @@ class PeerConnection {
   using IceCandidateCallbackPtr = std::function<void(IceCandidate)>;
   using DataChannelCallbackPtr = std::function<void(std::shared_ptr<DataChannel> channel)>;
 
-  PeerConnection(std::string stun_server, int stun_port, IceCandidateCallbackPtr icCB, DataChannelCallbackPtr dcCB, std::string sdp);
+  PeerConnection(std::string stun_server, int stun_port, IceCandidateCallbackPtr icCB, DataChannelCallbackPtr dcCB);
+
   virtual ~PeerConnection();
 
   /**
-   * Generate a local SDP (answer)
+   *
+   * Parse Offer SDP
+   */
+  void ParseOffer(std::string offer_sdp);
+  
+  /**
+   * Generate Answer SDP
    */
-  std::string GenerateSDP();
+  std::string GenerateAnswer();
```

Comment:
I like this; and it makes it clear where to add "GenerateOffer" and "ParseAnswer" in the future.